src/compiler/diagnosticMessages.json

```diff
@@ -1859,10 +1859,22 @@
         "category": "Error",
         "code": 2553
     },
-    "Expected {0} type arguments, but got {1}.": {
+    "Expected at least {0} arguments, but got {1}.": {
         "category": "Error",
         "code": 2554
     },
+    "Expected {0} arguments, but got a minimum of {1}.": {
+        "category": "Error",
+        "code": 2555
+    },
+    "Expected at least {0} arguments, but got a minimum of {1}.": {

```

Comment:
I used "a minimum of" for spreads to indicate that a spread adds 0 or more arguments to those already present. After reading the example you might have a better idea for the wording.

Here's an example that would give "a minimum of 1":

```ts
declare function f(first, second, ...rest): any[]
declare const car: number
declare const cdr: number[]
f(car, ...cdr) // Expected at least 2 arguments, but got a minimum of 1.
```

Note that this is legal Javascript and could succeed at runtime, but we can't prove that it will because `cdr` might be empty.